src/services/SalesEmployeeService.ts

```diff
@@ -0,0 +1,71 @@
+import axios, { AxiosResponse } from "axios";
+import { SalesEmployeeResponse } from "../models/SalesEmployeeResponse";
+import { SalesEmployeeRequest } from "../models/SalesEmployeeRequest";
+
+export const getSalesEmployees = async (): Promise<SalesEmployeeResponse[]> => {
+    try {
+        const response: AxiosResponse = await axios.get("http://localhost:8080/api/sales-employee");
```

Comment:
not sure i like this hardcoding, the url before /sales-employee should be abstracted